sklearn/preprocessing/data.py

```diff
@@ -1580,6 +1609,20 @@ class Normalizer(BaseEstimator, TransformerMixin):
         copy (if the input is already a numpy array or a scipy.sparse
         CSR matrix).
 
+    Examples
+    --------
+    >>> from sklearn.preprocessing import Normalizer
+    >>> X = [[ 1., -1.,  2.],
```

Comment:
Again we can choose numbers whose sum-of-squares is a nice square to make a denominator...

Reply:
done